nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -163,12 +164,12 @@ in {
 
     services.xserver.updateDbusEnvironment = true;
 
-    environment.variables.GIO_EXTRA_MODULES = [ "${lib.getLib pkgs.gnome3.dconf}/lib/gio/modules"
-                                                "${pkgs.gnome3.glib-networking.out}/lib/gio/modules"
+    environment.variables.GIO_EXTRA_MODULES = [ "${pkgs.gnome3.glib-networking.out}/lib/gio/modules"
                                                 "${pkgs.gnome3.gvfs}/lib/gio/modules" ];
     environment.systemPackages = pkgs.gnome3.corePackages ++ cfg.sessionPath
       ++ (removePackagesByName pkgs.gnome3.optionalPackages config.environment.gnome3.excludePackages) ++ [
       pkgs.xdg-user-dirs # Update user dirs as described in http://freedesktop.org/wiki/Software/xdg-user-dirs/
+      pkgs.gnome3.mutter # Give eg. gnome-control-center access to schema
```

Comment:
IIRC, mutter was required by g-c-c for keyboard panel, not schemas.

Reply:
It fetches (and sets) the keybindings from mutters schema. I can make the comment explicit that it's the keyboard panel functionality that require them.

Reply:
Oh, right. The keybindings XML files point to schemas. I think I have somewhere g-c-c patch that adds `schemafile` argument to `KeyListEntries` element from the [time I wanted to get rid of `sessionPath`](https://github.com/NixOS/nixpkgs/pull/43992).